src/main/java/org/folio/dataexp/service/logs/ErrorLogService.java

```diff
@@ -237,4 +236,7 @@ private ErrorLog getGeneralErrorLog(String errorMessageCode, UUID jobExecutionId
       .jobExecutionId(jobExecutionId);
   }
 
+  private String getInventoryRecordLink() {
+    return configurationService.getValue(INVENTORY_RECORD_LINK_KEY);
```

Comment:
Can this be cached value?

Reply:
@khandramai ,
configurationService uses ConfigurationRepository (extends JpaCqlRepository).
so, the value will be cached by the Hibernate by default.
I believe it's enough.

Do you believe, it's reasonable ?